test/Numerics/DGMethods/advection_diffusion/advection_diffusion_model_1dimex_bgmres.jl

```diff
@@ -307,7 +307,7 @@ let
                     β = FT(1 // 100)
                     μ = FT(-1 // 2)
                     δ = FT(1 // 10)
-
+                    connectivity = dim == 2 ? :face : :full
```

Comment:
Should connect `:full` and `:face` be the same for 2D Stacked meshes?

Reply:
Yes. :full is only set as default for StackedBrick and StackedCubedSphere meshes (2D extruded to 3D). Defaults are left in place for other cases.